src/base/bittorrent/session.cpp

```diff
@@ -1863,20 +1863,26 @@ TorrentHandle *Session::findTorrent(const InfoHash &hash) const
 
 bool Session::hasActiveTorrents() const
 {
-    foreach (TorrentHandle *const torrent, m_torrents)
-        if (TorrentFilter::ActiveTorrent.match(torrent))
-            return true;
-
-    return false;
+    return std::any_of(m_torrents.begin(), m_torrents.end(), [](TorrentHandle *torrent)
+    {
+        return TorrentFilter::ActiveTorrent.match(torrent);
+    });
```

Comment:
@Couchy:  Is there a reason for the `);` sequence, here and below?  

I am fairly certain existing code does not use it.

Reply:
The reason is C++ syntax.

Reply:
@zeul suggested it as a refactor in the first comment. I just changed these similar methods to match it.